scripts/start_civ.sh

```diff
@@ -50,6 +50,8 @@ GUEST_AAF_DIR=$WORK_DIR/aaf
 GUEST_AAF_CONFIG=$GUEST_AAF_DIR/mixins.spec
```

Comment:
We are planning to remove start_civ.sh. So please do not modify it anymore. Thanks.

Reply:
Thanks Yadong, we will switch to vm-manager, right? There is no difference between caac_dev and caas_cfc for LG start. So I think if we don't need start_civ.sh, then we don't need this changes any more.

Reply:
Yes, you are right, we will switch to vm-manager.

Reply:
Thanks Yadong. I have updated the patches, and remove start_civ.sh changes. 
If the setup_host.sh change is only for caas_cfc, maybe we need keep it under device-specific/caac_cfc folder? And write it to one separate script? Similar to PR: https://github.com/projectceladon/device-androidia-mixins/pull/1192

Reply:
yes, if you are comfortable to keep it under device-specific/caas_cfc. depends on you. :)

Reply:
Ok, thanks Yadong. I will keep this in setup host. Then we only need run setup host when we setup env. Instead of keep another script, : )